trace/trace.go

```diff
@@ -55,8 +86,8 @@ func writeTraceItemSummary(b *bytes.Buffer, msg string, totalTime time.Duration,
 	b.WriteString(fmt.Sprintf("%vms (%v)", durationToMilliseconds(totalTime), startTime.Format("15:04:00.000")))
 }
 
-func durationToMilliseconds(timeDuration time.Duration) int64 {
-	return timeDuration.Nanoseconds() / 1e6
+func durationToMilliseconds(timeDuration time.Duration) string {
```

Comment:
I think we can should on time.Duration String method. Please remove this function